packages/mdc-button/_mixins.scss

```diff
@@ -60,6 +60,10 @@
   border-radius: $corner-radius;
 }
 
+@mixin mdc-button-horizontal-padding($left, $right: $left) {
+  @include mdc-rtl-reflexive-property(padding, $left, $right);
```

Comment:
Don't use reflexive-property if both values are the same, it creates excess styles.